documentation/adoc/cluster-controller.adoc

```diff
@@ -135,6 +135,9 @@ data:
     }
 ----
 
+WARNING: If the cluster is deployed using the "ephemeral" storage, the Kafka brokers could have problems to deal with
+Zookeeper nodes restarts which could happen on configuration updates in the cluster ConfigMap.
+
```

Comment:
I think we need to word this a little differently because the problem will definitely happen when:

* ZK is using ephemeral storage and
* ZK has only 1 node and
* The ZK pod is rolled, or otherwise get recreated

As it's currently worded is "could" have problems, but we're not be explicit about exactly which configurations will have these problems.

Reply:
@tombentley even if I have a 3 ZK nodes it could happen when I change a CM and a rolling update is started. In any case the 3 ZK pods lose their state and Kafka brokers cannot re-connect to the new ones.

Reply:
@tombentley I don't think our rolling update is sophisticated enough to make sure that the same doesn't happen with 3 node ephemeral ZK cluster. We do not make sure that between the pod restarts the data are replicated. So we are 100% sure it happens with single node and we are XX% (==could) sure it happens with more nodes as well.